.github/workflows/release.yaml

```diff
@@ -9,7 +9,7 @@ on:
       bundle_channels:
         description: "Bundle channels"
         required: true
-        default: candidates
```

Comment:
let's not make this change -we're not using GH actions here anyway and it's just going to cause conflicts

Reply:
It's generated automatically by `make gen` so it would be overwritten after next sync job.

Reply:
ah, I see